src/main/java/core/basesyntax/fruitstoreoperation/ReturnFruitOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.fruitstoreoperation;
+
+import core.basesyntax.InputDataModel;
+
+public class ReturnFruitOperation implements FruitStoreOperation {
+    @Override
+    public void doOperation(InputDataModel product, int amount) {
+        SupplyFruitOperation supplyFruitOperation = new SupplyFruitOperation();
```

Comment:
do we need to create all time a new object when we call this method?
better write here its own logic

Reply:
done